keps/sig-scheduling/2249-pod-affinity-namespace-selector/README.md

```diff
@@ -0,0 +1,739 @@
+  users need to explicitly set the NamespaceSelector parameter in the pod spec,
+  if the feature is disabled the field is preserved if it was already set in the
+  presisted pod object, otherwise it is silently dropped.
+
+* **What specific metrics should inform a rollback?**
+  - A spike on metric `schedule_attempts_total{result="error|unschedulable"}`
+    when pods using this feature are added.
+  - A spike on `plugin_execution_duration_seconds{plugin="InterPodAffinity"}`.
+
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+No, will be manually tested.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
+fields of API types, flags, etc.?**
+  No.
+
+### Monitoring Requirements
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can an operator determine if the feature is in use by workloads?**
+  The operator can query pods with the NamespaceSelector field set in pod affinity terms.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine 
+the health of the service?**
+  - [x] Metrics
+    - Component exposing the metric: kube-scheduler
+      - Metric name: `pod_scheduling_duration_seconds`
+      - Metric name: `plugin_execution_duration_seconds{plugin="InterPodAffinity"}`
+      - Metric name: `schedule_attempts_total{result="error|unschedulable"}`
+  - [ ] Other (treat as last resort)
+    - Details:
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  - 99% of pod scheduling latency is within x minutes
+  - 99% of `InterPodAffinity` plugin executions are within x milliseconds
+  - x% of `schedule_attempts_total` are successful
+
+* **Are there any missing metrics that would be useful to have to improve observability 
+of this feature?**
+No.
+
+### Dependencies
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **Does this feature depend on any specific services running in the cluster?**
+No
+
+
+### Scalability
+
+_For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them._
+
+_For beta, this section is required: reviewers must answer these questions._
+
+_For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field._
+
+* **Will enabling / using this feature result in any new API calls?**
+No. 
+
+* **Will enabling / using this feature result in introducing new API types?**
+No.
+
+* **Will enabling / using this feature result in any new calls to the cloud 
+provider?**
+No.
+
+* **Will enabling / using this feature result in increasing size or count of 
+the existing API objects?**
+Yes, if users set the NamespaceSelector field.
+
+* **Will enabling / using this feature result in increasing time taken by any 
+operations covered by [existing SLIs/SLOs]?**
+No.
```

Comment:
Technically, scheduling (which is part of pod-startup time) may increase.
I know you have a plan to keep it under control, but it's probably worth mentioning anyway

Reply:
clarified.